src/openzaak/forms/fields.py

```diff
@@ -16,3 +16,8 @@ def __init__(self, *args, **kwargs):
         assert "empty_value" not in kwargs, "empty_value may not be provided"
         kwargs["empty_value"] = None
         super().__init__(*args, **kwargs)
+
+    def clean(self, value):
+        self.validate(value)
+        value = super().clean(value)
```

Comment:
```suggestion
        value = super().clean(value)
        self.validate(value)
```
I think self.validate expects the `value` which was already prepared, so it should be called after
In this particular case we don't have validators to the field, so it doesn't matter, but for the future let's switch the lines

Reply:
`self.validate(value)` must be first to check that the field is not empty. `RelativeDeltaFormField` does not use the validate function by default.